src/com/github/igotyou/FactoryMod/FactoryModManager.java

```diff
@@ -29,13 +29,17 @@
 	private HashSet<Material> possibleInteractionBlock;
 	private Material factoryInteractionMaterial;
 	private boolean citadelEnabled;
+	private int redstonePowerOn;
+	private int redstoneRecipeChange;
 	private String compactLore;
 
 	public FactoryModManager(FactoryModPlugin plugin,
-			Material factoryInteractionMaterial, boolean citadelEnabled) {
+			Material factoryInteractionMaterial, boolean citadelEnabled, int redstonePowerOn, int redstoneRecipeChange) {
```

Comment:
I think this would be better as a static in the interaction manager, because any new factory thats not the classic FurnaceCraftingtableChest setup wont need those values for anything, so they dont fit in the manager and they are not used anywhere outside of the interaction manager class. 

It could just have a hardcoded default and a static method to adjust it, if the config specifies it.